Description: GCC 6 compilation fixes
Author: Stephen Kitt <skitt@debian.org>

Add a missing header.
Propagate packed attributes as necessary.

--- a/lib/xstrduplwr.c
+++ b/lib/xstrduplwr.c
@@ -25,6 +25,7 @@
 
 #include <sys/types.h>
 #include <ctype.h>
+#include <string.h>
 
 #include "xalloc.h"
 #include "xstrduplwr.h"
--- a/src/pcx.h
+++ b/src/pcx.h
@@ -23,7 +23,7 @@
 
 #include "video.h"
 
-typedef struct pcx_header_type
+typedef struct ATTRIBUTE_PACKED pcx_header_type
 {
   a_u8 signature;
   a_u8 version;
@@ -39,9 +39,9 @@
   a_u16 palette_kind;
   a_u8 rien[58];
 }
-a_pcx_header ATTRIBUTE_PACKED;
+a_pcx_header;
 
-typedef struct
+typedef struct ATTRIBUTE_PACKED
 {
   a_u8 r, g, b;
 }
@@ -54,7 +54,7 @@
 }
 a_palette;
 
-typedef struct img_type
+typedef struct ATTRIBUTE_PACKED img_type
 {
   a_pcx_header header;
   a_palette palette;
